src/rust/ide/view/src/project.rs

```diff
@@ -167,6 +175,7 @@ impl View {
         let graph                      = &model.graph_editor.frp;
         let network                    = &frp.network;
         let searcher_left_top_position = Animation::<Vector2<f32>>::new(network);
+        model.set_style(false);
```

Comment:
the api is cryptic. I know it was before, but lets make it better. Lets do `model.set_style(theme::builtin::Light)` instead. The `theme::builtin` module will be available after merging my PR today night.